image-loader/app/controllers/ImageLoaderController.scala

```diff
@@ -88,9 +89,7 @@ class ImageLoaderController(auth: Authentication,
             case e: UnsupportedMimeTypeException => FailureResponse.unsupportedMimeType(e, config.supportedMimeTypes)
             case e: ImageProcessingException => FailureResponse.notAnImage(e, config.supportedMimeTypes).as(ArgoMediaType)
             case e: java.io.IOException => FailureResponse.badImage(e).as(ArgoMediaType)
-            case e =>
-              logger.error("Failed upload", e)
-              InternalServerError(Json.obj("error" -> e.getMessage)).as(ArgoMediaType)
```

Comment:
Not sure if I missed this last time, but this log line seems useful to me. Otherwise we do not log the full exception of the problem preventing file upload (the next line only extracts the message from the exception).